WebRTCPlugin/VideoRenderer.cpp

```diff
@@ -28,9 +30,26 @@ void CVideoRenderer::OnFrame(const webrtc::VideoFrame& frame)
 		variant_t height = videoWidth;
 		DispatchAsync(onresize, width, height);
 	}
-
-	//Clone video frame
-	auto clone = new webrtc::VideoFrame(frame);
+	
+	if (frame.rotation() != rotation)
+		RTC_LOG(LS_INFO) << "frame rotation changed to " << frame.rotation();
+	rotation = frame.rotation();
+
+	// Only iOS rotated frames needs to be adjusted.
+	webrtc::VideoFrame* clone = nullptr;
+	if (rotation == webrtc::kVideoRotation_90 ||

```

Comment:
I think you need treat kVideoRotation_90 case differently than kVideoRotation_180. In kVideoRotation_90 you need also swap between width and height. Currently plugin stretch iPhone video

Reply:
Finished yesterday.